### PR TITLE
Orbit Yaw Parameter

### PR DESCRIPTION
**Pull request by Claudio-Chies**
Relevant PR:
Updated the documentation to reflect the changes proposed in

- [ ] https://github.com/PX4/PX4-Autopilot/pull/23358

Cleared up some errors whereby Parameters for the Orbit existed but were not mentioned.

### CODE REVIEW

File: en/advanced_config/parameter_reference.md [hunk 1]
```diff
@@ -12950,6 +12950,13 @@ table {
  <td>1000.0</td>
  <td>m</td>
 </tr>
+<tr>
```
Comment (hamishwillee) [inline en/advanced_config/parameter_reference.md:12953]: Please remove changes to this file. The parameters are updated from PX4-Autopilot automatically - so after your PR merges we'll do a rebase and get this.

Comment (hamishwillee) [reply en/advanced_config/parameter_reference.md:12953]: Actually, I'll do it

**Comment by hamishwillee** [comment]
OK, this looks great. I removed the parameter change and ran prettier. 

One more thing, is this going into PX4 v1.15 or main? Depending on which, we need to add a release note to either https://docs.px4.io/main/en/releases/1.15.html or https://docs.px4.io/main/en/releases/main.html - can you do that?

File: en/advanced_config/parameter_reference.md (hunk 1 above)
Comment (Claudio-Chies) [reply en/advanced_config/parameter_reference.md]: good to know, thanks!

**Comment by hamishwillee** [comment]
@Claudio-Chies Before this goes it it will need that release note in the main branch - can you do that?

**Comment by Claudio-Chies** [comment]
@hamishwillee updated the release notes as requested. but we will have to wait until https://github.com/mavlink/mavlink/pull/2132 gets sorted beforce merging

File: en/releases/main.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 ### Multi-Rotor
 
 - TBD
```
Comment (hamishwillee) [inline en/releases/main.md:67]:
```suggestion
```

File: en/releases/main.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 
 ### Multi-Rotor
 
-- TBD
+- Added vehicle specific yaw behavior for orbit flight mode, which can be set via an [Parameter](../advanced_config/parameter_reference.md#MC_ORBIT_YAW_MOD) ([PX4-Autopilot#23358](https://github.com/PX4/PX4-Autopilot/pull/23358))
```
Comment (hamishwillee) [inline en/releases/main.md:67]: Propose this as a little more clear.

```suggestion
- Allow system-default [multicopter orbit mode](../flight_modes_mc/orbit.md) yaw behaviour to be configured, using the parameter [MC_ORBIT_YAW_MOD](../advanced_config/parameter_reference.md#MC_ORBIT_YAW_MOD) ([PX4-Autopilot#23358](https://github.com/PX4/PX4-Autopilot/pull/23358))
```

**Comment by hamishwillee** [comment]
@Claudio-Chies Link needs to be fixed, but this can be merged right?

**Comment by Claudio-Chies** [comment]
Yea, and i need to wait for the qgroundcontrol PR, i'll try to do it today

**Comment by Claudio-Chies** [comment]
> @Claudio-Chies Link needs to be fixed, but this can be merged right?

@hamishwillee  actually, i dont know where LinkedFileMissingAnchor comes from, i'm not changing or referencing anything in airframe_reference.md

**Comment by hamishwillee** [comment]
@Claudio-Chies I rebased against main and the flaw disappeared. So I suspect this was never in your code.

Ping me when you want this merged (I am taking your comment above to mean you are waiting on something in QGC)

**Comment by Claudio-Chies** [comment]
@hamishwillee 
now its ready to be merged, QGC got merged

**Comment by hamishwillee** [comment]
Done. Yay. Congratulations :-)

**Pull request merged**